pwiz_tools/Skyline/TestFunctional/DetectionsPlotTest.cs

```diff
@@ -78,8 +78,9 @@ protected override void DoTest()
             RunUI(() =>
             {
                 propDialog.SetQValueTo(0.003f);
+                propDialog.OkDialog();
             });
-            OkDialog(propDialog, propDialog.OkDialog);
+            WaitForClosedForm(propDialog);

```

Comment:
This is functionally equivalent to the code it replaces.

Reply:
Saves one Invoke() call. My initial theory was that two invoke calls in quick succession might occasionally be executed out of order.  It proved to be incorrect, but since it's slightly more efficient than the original code I decided to keep it.